extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -33,13 +35,18 @@ const buildSliderTemplate = (element) => {
     <div class="i-amphtml-story-interactive-slider-container">
       <div class="i-amphtml-story-interactive-prompt-container"></div>
       <div class="i-amphtml-story-interactive-slider-input-container">
-        <input
-          class="i-amphtml-story-interactive-slider-input"
-          type="range"
-          min="0"
-          max="100"
-          value="25"
-        />
+        <div class="i-amphtml-story-interactive-side-values">0</div>
```

Comment:
The left and right numbers are an optional display type and should be removed for this first PR.
<img width="479" alt="Screen Shot 2021-07-15 at 12 09 08 PM" src="https://user-images.githubusercontent.com/3860311/125820939-e978aa6c-42f8-4a53-a322-a28aa6b6a45a.png">

The first PR needs to focus on the default UI type, that way we can build on top of the default.
We want to make sure all of our margins work out-of-the-box and do not need to be overridden.